teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -66,21 +69,24 @@
   @Option(
       names = {"--p2p-advertised-ip"},
       paramLabel = "<NETWORK>",
-      description = "P2P advertised IP",
+      description = "P2P advertised IP (Default: 127.0.0.1)",
       arity = "1")
   private String p2pAdvertisedIp;
 
   @Option(
       names = {"--p2p-advertised-port"},
       paramLabel = "<INTEGER>",
-      description = "P2P advertised port",
+      description = "P2P advertised port. The default is the port specified in --p2p-port",
       arity = "1")
   private Integer p2pAdvertisedPort;
 
   @Option(
       names = {"--p2p-private-key-file"},
       paramLabel = "<FILENAME>",
-      description = "This node's private key file",
+      description =
+          "This node's private key file. If one is not specified, the system looks for a previously generated key in the key-value store "
+              + "(i.e. <beacon-data-dir>/kvstore/generated-node-key.dat) and uses that if available. "
+              + "If not, then a new key is generated for use and persisted in the key-value store.",
```

Comment:
We probably don't want to expose the inner details of how the beacon node store's its data.  So maybe:
```suggestion
          "This node's private key file. If not specified, uses or generates a key which is stored within the <beacon-data-dir>.",
```

Reply:
@bgravenorst Any chance you have a nice succinct way of phrasing this?

Reply:
This is a bit wordier, but how about, "The node's private key file. Defaults to the key file in <beacon-data-dir>. If no key file exists, Teku creates a key file containing the generated private key."?

Reply:
> We probably don't want to expose the inner details …

makes sense. 

@ajsutton's suggestion feels a bit more concise, though @bgravenorst's communicates the sequence of default behaviors a bit more clearly. I might apply @ajsutton's suggestion for now, and we can change it if you feel different. I guess we can probably be more descriptive in the CLI reference docs instead if we feel there is still room for ambiguity.